packages/fsweb/proxy/server.prod.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const Proxy = require('./proxy');
+
+const envPath = process.env.ENV_PATH || '../../env';
+const env = require(
+  path.resolve(__dirname, '../../', envPath)
+);
+const port = process.env.PORT || 3000;
+
+if (env && env.dataSource.type === 'commercecloud') {
+  console.log('Env not found in path, ', envPath);
+
+  const proxy = new Proxy(env.dataSource.apiConfig).initProxy();
+
+  if (!proxy) {
+    return;
+  }
+
+  const index = path.resolve(__dirname, '..', 'web', 'build', 'index.html');
```

Comment:
This needs to be the compiled version, because the bundle.js file needs to be injected.

Reply:
@Cauldrath done